packages/edit-site/src/components/dataviews/in-filter.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalInputControlPrefixWrapper as InputControlPrefixWrapper,
+	SelectControl,
+} from '@wordpress/components';
+
+export default ( { id, fields, view, onChangeView } ) => {
+	const field = fields.find( ( f ) => f.id === id );
+
+	return (
+		<SelectControl
+			value={ view.filters[ id ] }
+			prefix={
+				<InputControlPrefixWrapper
+					as="span"
+					className="dataviews__select-control-prefix"
+				>
+					{ field.header + ':' }
+				</InputControlPrefixWrapper>
+			}
+			options={ field?.elements || [] }
+			onChange={ ( value ) => {
+				if ( value === '' ) {
+					value = undefined;
+				}
+
+				onChangeView( ( currentView ) => ( {
+					...currentView,
+					filters: {
```

Comment:
Maybe can include a cleanEmptyObject call to remove the filter property if there are no filters at all.

Reply:
https://github.com/WordPress/gutenberg/pull/55270/commits/4bd098767ab009c6e35b87031a50407aca51ec44